bve-client/src/main.rs

```diff
@@ -240,16 +238,17 @@ fn main() {
 
             camera_location = camera_location.add_element_wise(dir_vec);
 
-            renderer.set_camera_location(camera_location);
+            block_on(async { client.lock().await.renderer.set_camera_location(camera_location) });
```

Comment:
Block on is just a way to do an async operation in a sync context. That client handle is used in async contexts as well where it being async is a good thing. In this case it's just an async lock which is usable in both async and non-async contexts through block_on. The amount of block_on will go down as things evolve, but just because I'm blocking to do an operation in one particular place doesn't mean that that's a bad idea as a whole.

Reply:
That's fair